packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -689,6 +689,12 @@ export function determineStates(previous: ComplianceState, current: boolean | un
   }
 }
 
+// The debounce time for the stopped state to be shown after both states are stopped.
+// This is to prevent stopped messages from being lost by transitioning to "Off" too
+// quickly if the states are toggled in quick succession.
+// This also prevents React strict mode from transitioning to "Off" too quickly.
+const ComplianceNotificationOffDebounceTimeMs = 2000;
```

Comment:
Is this time limit specified by SDK or are we deciding it

Reply:
We are deciding it from a UI perspective. Essentially an amount of time that we can presume that it would be OK to clear the previous message and move the state to the next stage if another update came in